examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -739,13 +739,22 @@ void WindowManager::UpdateLCD()
         Attributes::CurrentPositionTilt::Get(cover.mEndpoint, tilt);
         chip::DeviceLayer::PlatformMgr().UnlockChipStack();
 
-#ifdef DISPLAY_ENABLED
```

Comment:
Why do we remove the ifdef? It seems ok to not call the LCD paint if the display is not enabled

Reply:
This check is duplicate one as the same ifdef is present just above in the code. (Line #724)

Reply:
ah thank you!